<?xml version="1.0" encoding="UTF-8"?><testrun name="TestControl" project="blackjack" tests="7" started="7" failures="1" errors="0" ignored="0">
  <testsuite name="test.TestControl" time="1.122">
    <testcase name="testControl1" classname="test.TestControl" time="1.053"/>
    <testcase name="testControl2" classname="test.TestControl" time="0.007"/>
    <testcase name="testControl3" classname="test.TestControl" time="0.008"/>
    <testcase name="testControl4" classname="test.TestControl" time="0.009"/>
    <testcase name="testControl5" classname="test.TestControl" time="0.035">
      <failure>
        <expected>hello J</expected>
        <actual>hello Q</actual>junit.framework.ComparisonFailure: expected:&lt;hello [J]&gt; but was:&lt;hello [Q]&gt;&#13;
	at junit.framework.Assert.assertEquals(Assert.java:81)&#13;
	at junit.framework.Assert.assertEquals(Assert.java:87)&#13;
	at test.TestControl.testControl5(TestControl.java:86)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)&#13;
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)&#13;
	at java.lang.reflect.Method.invoke(Method.java:606)&#13;
	at junit.framework.TestCase.runTest(TestCase.java:164)&#13;
	at junit.framework.TestCase.runBare(TestCase.java:130)&#13;
	at junit.framework.TestResult$1.protect(TestResult.java:106)&#13;
	at junit.framework.TestResult.runProtected(TestResult.java:124)&#13;
	at junit.framework.TestResult.run(TestResult.java:109)&#13;
	at junit.framework.TestCase.run(TestCase.java:120)&#13;
	at junit.framework.TestSuite.runTest(TestSuite.java:230)&#13;
	at junit.framework.TestSuite.run(TestSuite.java:225)&#13;
	at org.eclipse.jdt.internal.junit.runner.junit3.JUnit3TestReference.run(JUnit3TestReference.java:130)&#13;
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:467)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:683)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:390)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:197)&#13;
&#13;
</failure>
    </testcase>
    <testcase name="testControl6" classname="test.TestControl" time="0.004"/>
    <testcase name="testControl7" classname="test.TestControl" time="0.005"/>
  </testsuite>
</testrun>
